src/cgame/cg_rocket_draw.cpp

```diff
@@ -1365,8 +1362,8 @@ class LagometerElement : public HudElement
 public:
 	LagometerElement( const Rml::String& tag ) :
 			HudElement( tag, ELEMENT_GAME, true ),
-			shouldDrawLagometer( true ),
-			adjustedColor( Color::White )
+			shouldDrawLagometer_( true ),
+			adjustedColor_( Color::Alpha )
```

Comment:
There's a mistake here in the trailing underscore commit (`Color::Alpha`)

Reply:
I had fixed that but included the fix in the wrong commit. Did the magic to correct the first commit.